python/lsst/pipe/supertask/examples/calexpToCoaddTask.py

```diff
@@ -0,0 +1,66 @@
+"""Simple example SuperTask for testing purposes.
+"""
+
+import lsst.log as lsstLog
+from lsst.pex.config import ConfigField
+from lsst.pipe import supertask
+from lsst.pipe.base.struct import Struct
+
+_LOG = lsstLog.Log.getLogger(__name__)
+
+
+class CalexpToCoaddTaskConfig(supertask.SuperTaskConfig):
+    calexp = ConfigField(dtype=supertask.InputDatasetConfig,
+                         doc="DatasetType for the input image")
+    coadd = ConfigField(dtype=supertask.OutputDatasetConfig,
+                        doc="DatasetType for the output image")
+
+    def setDefaults(self):
+        # set units of a quantum, this task uses per-tract-patch-filter quanta
+        self.quantum.units = ["SkyMap", "Tract", "Patch", "AbstractFilter"]
+        self.quantum.sql = None
```

Comment:
Just an aside: am I correct in thinking that we would have no way to use this if it was not `None`?  Do you have any plans for how you *would* incorporate per-SuperTask SQL statements?